public/app/screens/main/sections/settings/user-avatar/style.scss

```diff
@@ -0,0 +1,28 @@
+@import "theme";
+
+.cc-user-avatar{
+  @include grid-row(collapse);
+
+  position: relative;
+  margin: convert-to-rem(8px);
+
+  .cc-user-avatar__image-content {
+    @include grid-column(10);
+    .cc-user-avatar__image{
+      width: convert-to-rem(85px);
```

Comment:
We said that we are going to use `rem` dimension only for paddings, margins (?) and font sizes (?)